dashboard/app/controllers/courses_controller.rb

```diff
@@ -192,20 +192,19 @@ def course_params
     cp = params.permit(:version_year, :family_name, :has_verified_resources, :has_numbered_units, :pilot_experiment, :published_state, :announcements).to_h
     cp[:announcements] = JSON.parse(cp[:announcements]) if cp[:announcements]
 
+    cp[:published_state] = SharedConstants::PUBLISHED_STATE.beta unless cp[:published_state]
```

Comment:
I agree with you in the long run we will want the default published_state to be `in_development`. We haven't actually built on the ability for something to be in that state yet because there is no easy way to do that with the existing set of booleans that controls this logic. So I was starting by moving everything over to published_state and then adding the ability for something to be in_development after.

Reply:
ahh, makes sense! this answers several of my questions, apparently :-)